unfetter-discover/unfetter-ui
# Load report url

supports unfetter-discover/unfetter#340
requires 
* https://github.com/unfetter-discover/unfetter/pull/606
* https://github.com/unfetter-discover/unfetter-store/pull/67

## To Test
* Pull PRs for 3 repos
* Restart stack
* Visit threat dashboard
* Click Add FAB button
* Verify new 'load by url' step in stepper
* enter an incorrect url, notice the error msg (have to click back at the moment)
* enter a correctly formatted url (any will do), notice the stepper auto populates form based on sample rules in config collection
* click save and close
* verify screen refreshes, state looks good

### Note:
* There is a known issue with the kill chain reordering after save and refresh
* Also verify the malware step in the add stepper is a multiselect list, preserving preselected values
* Also not the collapsible side panel
* ChangeDetectionStrategy was set to be less aggressive and should help performance
